Fix type annotation

https://github.com/sgeisler/termgraph/blob/0e0fa4aa9754ee7b58fe957aee9f7807b43c5ddf/examples/__init__.py#L5

According to Pylance in [Visual Studio Code](https://code.visualstudio.com/), I get the following tooltip when editing `examples/__init__.py`

```
List expression not allowed in type annotation
  Use List[T] to indicate a list type or Union[T1, T2] to indicate a union type
```

Running [mypy](https://github.com/python/mypy) also show the problem with

```
termgraph\__init__.py:2: error: Skipping analyzing 'colorama': found module but no type hints or library stubs
termgraph\__init__.py:2: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
termgraph\__init__.py:56: error: Bracketed expression "[...]" is not valid as a type
termgraph\__init__.py:56: note: Did you mean "List[...]"?
termgraph\__init__.py:73: error: Incompatible types in assignment (expression has type "float", variable has type "int")
examples\__init__.py:5: error: Bracketed expression "[...]" is not valid as a type
examples\__init__.py:5: note: Did you mean "List[...]"?
Found 4 errors in 2 files (checked 7 source files)
```

I'm not a specialist of type annotation and Python but https://docs.python.org/3/library/typing.html and https://github.com/typeddjango/awesome-python-typing provide a lot of information.

type linting may be considered during CI builds

linting more generally (with tools such as [flake8](https://flake8.pycqa.org/en/latest/)) also

--- a/examples/__init__.py
+++ b/examples/__init__.py
@@ -1,8 +1,9 @@
 import csv
 from termgraph import Candle
+from typing import List
 
 
-def load_data(file: str) -> [Candle]:
+def load_data(file: str) -> List[Candle]:
     with open(file) as csvfile:
         reader = csv.reader(csvfile, delimiter=",")
         next(reader)  # skip first row

--- a/termgraph/__init__.py
+++ b/termgraph/__init__.py
@@ -1,5 +1,6 @@
 from math import ceil, floor
 from colorama import Fore, Style
+from typing import List
 
 __version__ = '0.1.0'
 
@@ -53,7 +54,7 @@ class CandleStickGraph:
     COLOR_POSITIVE = Style.BRIGHT + Fore.GREEN
     COLOR_NEGATIVE = Style.BRIGHT + Fore.RED
 
-    def __init__(self, data: [Candle], height: int):
+    def __init__(self, data: List[Candle], height: int):
         self._height = height
         self._data = data
 

